api/http/handler/registries/registry_create_test.go

```diff
@@ -81,7 +82,12 @@ func (t testRegistryService) Registries() ([]portainer.Registry, error) {
 	return nil, nil
 }
 
-func TestHandler_registryCreate(t *testing.T) {
+func (t testRegistryService) Create(registry *portainer.Registry) error {
+	return nil
+}
+
+// Not entirely sure what this is intended to test
+func deleteTestHandler_registryCreate(t *testing.T) {
```

Comment:
to test that registry is created?

Reply:
you'd think that from the name, but its making a payload struct, then making a local handler store function that saves that struct, and then tests that the struct == the struct.
from what i can read, the author mocked so much of Portainer out of the way, to not be testing other than the existance of a handler - which I think means its testing the handler code's calling of json unmarshaling?